app/services/idv/session.rb

```diff
@@ -99,6 +99,11 @@ def vendor_params
       applicant.merge('uuid' => current_user.uuid)
     end
 
+    def profile_id=(value)
+      session[:profile_id] = value
+      @profile = nil
+    end
```

Comment:
Bust the idv_session profile cache when in-person flow creates a new profile. @matthinz had to do the same thing in #9630.